Modules/TOF/include/Base/Counter.h

```diff
@@ -106,6 +108,18 @@ void Counter<size, labels>::Count(const unsigned int& v)
   counter[v]++;
 }
 
+template <const unsigned int size, const char* labels[size]>
+void Counter<size, labels>::Add(const unsigned int& v, const unsigned int& weight)
```

Comment:
@ercolessi why not making count derive from `Add(v, 1)` ?

Reply:
@njacazio cool, will do!